requestid/requestid.go

```diff
@@ -0,0 +1,36 @@
+package requestid
+
+import (
+	"context"
+	"github.com/google/uuid"
+	"github.com/infobloxopen/atlas-app-toolkit/gateway"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/metadata"
+)
+
+// DefaultRequestIDKey is the metadata key name for request ID
+var DefaultRequestIDKey = "Request-Id"
+
+func HandleRequestId(ctx context.Context) (reqId string) {
```

Comment:
Please add documentation to the public functions you added. (use `golint` to make sure your code in desirable quality)